dbms/src/Storages/Page/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 add_subdirectory(V2)
+add_subdirectory(tools)
```

Comment:
I think we should put these files into `dbms/src/Server/PageTools`

Reply:
`dtworkload` is in `Storages/DeltaMerge`, so I think put these files into `Storages/Page` is fine.